### ISSUE TITLE
expose ChipsAdapter mChipsAdapter

### ISSUE THREAD

**Issue by renannery** [pchmn/MaterialChipsInput#4]
I think is a good idea to make it public or even just create a getter method for `mChipsAdapter`, it'll make easier to customize the current adapter and have more control on the current list of chips.

How do you feel about that?

**Comment by pchmn** [comment]
Yeah why not, it's a good idea. Or even just propose a method to override the `onBindViewHolder()` of the adapter ?
